sympy/stats/tests/test_joint_rv.py

```diff
@@ -199,9 +191,24 @@ def test_NegativeMultinomial():
     raises(ValueError, lambda: NegativeMultinomial('b2', k0, 0.5, 0.4, 0.3, 0.4))
 
 
+def test_Wishart():
+    n, p = symbols('n p')
+    _k = Dummy('k')
+    A = MatrixSymbol('A', p, p)
+    V = MatrixSymbol('V', p, p)
+    X = Wishart('X', n, V)
+    assert str(density(X)(A)) == ("2**(-n*p/2)*pi**(-p*(p - 1)/4)*exp(-Trace(V**(-1)*A)/2)*"
```

Comment:
As pointed in other PRs, string comparison should be avoided and `dummy_eq` should be used.

Reply:
@sidhantnagpal I have tried using `dummy_eq` but there doesn't seem to be way to get it correct. So I have used str.

Reply:
There will be a way to get it correct. There are 4 things that can trip you up here:

1. Symbols are only equal if they have the same assumptions.
2. `dummy_eq` only considers Dummy symbols to be equal so you need to ensure that Dummy symbols are used in the same places in the two things you compare.
3. Some special kinds of Symbol (like MatrixSymbol and RandomSymbol) have an underlying symbol object that prints the same but compares unequal (e.g. `A != A.args[0]`)
4. Integer division in the repr needs to be fixed to e.g. S(1)/2.